src/seedu/addressbook/parser/Parser.java

```diff
@@ -11,17 +11,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.addressbook.commands.AddCommand;
-import seedu.addressbook.commands.ClearCommand;
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```

Comment:
Try not to use wildcard imports because this violates the coding style standard. Please use single imports and change your settings in Intellij: `Settings` > `Editor` > `Code Style` > `Java` > `Imports` and set both class count and names count to 99 (or any large number)